backend/src/main/java/com/google/backend/servlets/AlertsDataServlet.java

```diff
@@ -48,4 +59,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(new Gson().toJson(alertList));
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) 
+  throws ServletException, IOException {
+    // Read from the request body, which contains data in the format "alertId statusToChangeTo".
+    // For example, the body could be "4785074604081152 RESOLVED".
+    BufferedReader reader = request.getReader();
+    String body = reader.readLine();
+    if (body == null) {
+      log.warning(EMPTY_BODY_ERROR);
+      throw new ServletException(EMPTY_BODY_ERROR);
+    }
+    String[] data = body.split(" "); 
+    Long id = Long.parseLong(data[0]);
+    String status = data[1];
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    try {
+      Entity alertEntity = datastore.get(KeyFactory.createKey(Alert.ALERT_ENTITY_KIND, id));
+      alertEntity.setProperty(Alert.STATUS_PROPERTY, status);
+      datastore.put(alertEntity);
```

Comment:
ooc, you never deleted the old `alertEntity`, and here you put the `alertEntity` back in the datastore, will there be any duplication? will the altered `alertEntity` that you put back in the datastore replace the old `alertEntity`?

Reply:
yup I'm pretty sure that's how Datastore works!
https://cloud.google.com/datastore/docs/concepts/entities#creating_an_entity